README.md

```diff
@@ -34,6 +34,35 @@ python3 -m pip install .
 
 To use from your Python application, declare `awscrt` as a dependency.
```

Comment:
trivial: link the "advanced" options from up here in the "simple" build instructions...

Just replace this line about dependencies. Users that need to be told this, would also need us to show them how to do it, this isn't a library for users who need that much hand holding

```suggestion
See [Advanced Build Options](#advanced-build-options) for more info about building from source.
```